cfgrammar/src/lib/yacc/ast.rs

```diff
@@ -11,48 +11,50 @@ use super::{
 };
 
 use crate::Span;
-
-pub struct ASTValidation {
+/// Contains a `GrammarAST` structure produced from a grammar source file.
+/// As well as any errors which occurred during the construction of the AST.
+pub struct ASTWithValidityInfo {
     pub ast: GrammarAST,
     pub errs: Vec<YaccGrammarError>,
 }
 
-/// Provides access to an ast which may or may not be valid,
-/// An AST which is invalid is provided for further analysis.
-impl ASTValidation {
-    /// The AST will be invalid if errors were encountered during
-    /// construction in which case `errors()` will be non-empty.
+impl ASTWithValidityInfo {
+    /// Parses a source file into an AST, returning an ast and any errors that were
+    /// encountered during the construction of it.  The `ASTWithValidityInfo` can be
+    /// then usued to construct a `YaccGrammar`, which will either produce an
```

Comment:
s/usued/used/

Reply:
should be fixed in [9947d5e](https://github.com/softdevteam/grmtools/pull/354/commits/9947d5e559cd265de3f6265ff777db7db15a0c5f)